CarpetCleaning/StoredProcedures/Dbo.Loadexceldata.sql

```diff
@@ -0,0 +1,93 @@
+CREATE PROCEDURE dbo.LoadExcelData
+AS
+BEGIN
+
+WHILE ((SELECT COUNT(*) FROM ExcelData WHERE LoadDataNotes = 'Passed Validation') > 0)
+BEGIN
+
+	DECLARE @ThisRowId INT, 
+			@AddressID INT,
+			@JobID INT
+
+	DECLARE @CarpetCleaningServiceTypeId INT = 1,
+			@UrineTreatmentServiceTypeId INT = 2,
+			@ProtectantServiceTypeId INT = 3,
+			@UpholsteryServiceTypeId INT = 7,
+			@AirDuctsServiceTypeId INT = 9,
+			@TileServiceTypeId INT = 12,
+			@DryerVentServiceTypeId INT = 11,
+			@OtherServiceTypeId INT = 18
+				
+		--Inserts FROM the top row and works its way down.
+		SELECT TOP 1 @ThisRowId = RowId FROM dbo.Exceldata WHERE LoadDataNotes = 'Passed Validation'
```

Comment:
This select and all the rest that  come after should all be on the same level as these declares, no reason to indent any more